src/pages/resultsView/mutation/Mutations.tsx

```diff
@@ -238,6 +242,33 @@ export default class Mutations extends React.Component<
         }
     }
 
+    @autobind
+    @action
+    protected getDownloadData(dataType?: DataType): string {
+        var flatdata: any = '';
+        if (
+            this.selectedGene &&
+            this.props.store.getMutationMapperStore(this.selectedGene)
+        ) {
+            var mutationMapperStore: ResultsViewMutationMapperStore;
+            mutationMapperStore = this.props.store.getMutationMapperStore(
+                this.selectedGene
+            )!;
+            let mutations = mutationMapperStore.getMutationData();
+            let downloadData: any[] = [];
+            let sidToStudy = mutationMapperStore.studyIdToStudy.result!;
+            mutations.forEach(m => {
+                let study: any;
+                study = sidToStudy[m.studyId];
```

Comment:
```suggestion
                const study = sidToStudy[m.studyId];
```

Reply:
onur is right, you can just use type inference here.   in general, try to avoid using 'any' at all costs.  there are sometimes it can't be avoided or is very convenient.  but it should be used sparingly.